prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -353,16 +326,19 @@ public void destroy() throws FalconException {
         public void run() {
             try {
                 if (MONITORING_JDBC_STATE_STORE.getAllMonitoredEntities().size() > 0) {
-                    checkPendingInstanceAvailability(EntityType.FEED.toString());
-                    checkPendingInstanceAvailability(EntityType.PROCESS.toString());
+                    checkPendingInstanceAvailability();
 
                     // add Instances from last checked time to 10 minutes from now(some buffer for status check)
                     Date newCheckPointTime = new Date(now().getTime() + lookAheadWindowMillis);
                     addPendingEntityInstances(EntityType.FEED.toString(), null, newCheckPointTime);
```

Comment:
Can we change this also on the lines of checkPendingInstanceAvailability(), so we don't call this method twice?